SPEC.md

```diff
@@ -205,26 +215,26 @@ The test index makes use of symbolic language in describing connection and messa
 
 ### ZG-CONFORMANCE-016
 
-    Deploy a network of three nodes: Node 1, Node 2 and Node 3.
+    Deploy a multi-node network setup with nodes 1, 2 and 3.
     Connect a synthetic node to Node 1.
-    After squelching validator keys belonging to Node 2 and Node 3,
-    Node 1 shouldn't broadcast mtPROPOSE_LEDGER messages from squelched validators to the synthetic node.
+
+    Let A be a list of node public keys 1, 2 and 3.
+    Let B be a list of node public keys 2 and 3.
+
     <> with Node1
-    <- mtPROPOSE_LEDGER with the Node 1's node_pub_key1
-    <- mtPROPOSE_LEDGER with the Node 2's node_pub_key2
-    <- mtPROPOSE_LEDGER with the Node 3's node_pub_key3
-    -> mtSQUELCH (node_pub_key2)
-    -> mtSQUELCH (node_pub_key3)
-    <- mtPROPOSE_LEDGER with the Node 1's node_pub_key1
-    <- mtPROPOSE_LEDGER with the Node 1's node_pub_key1
-    <- mtPROPOSE_LEDGER with the Node 1's node_pub_key1
+    << mtPROPOSE_LEDGER (A)
+    -> mtSQUELCH (B)
+    << mtPROPOSE_LEDGER (A)
+
+    Assert: A synthetic node receives only mtPROPOSE_LEDGER messages with a key from node 1
+    after squelching node public keys belonging to nodes 2 and 3 (B).
 
 ### ZG-CONFORMANCE-017
-    
+
     The node sends mtMANIFESTS after the handshake.
```

Comment:
```suggestion
    The node sends mtMANIFESTS after the handshake.
    
```

There are many test cases where this space is missing, I think it makes the cases more readable.

Reply:
Done everywhere